src/components/hv-view/index.js

```diff
@@ -41,142 +47,213 @@ export default class HvView extends PureComponent<HvComponentProps> {
-        );
-        const defaultScrollToInputAdditionalOffset = 120;
-        if (scrollToInputAdditionalOffset) {
-          const parsedOffset = parseInt(scrollToInputAdditionalOffset, 10);
-          props.scrollToInputAdditionalOffset = Number.isNaN(parsedOffset)
-            ? 0
-            : defaultScrollToInputAdditionalOffset;
-        } else {
-          props.scrollToInputAdditionalOffset = defaultScrollToInputAdditionalOffset;
-        }
-
-        props.keyboardOpeningTime = 0;
-        props.keyboardShouldPersistTaps = 'handled';
-        props.automaticallyAdjustContentInsets = false;
-        props.scrollEventThrottle = 16;
-        props.getTextInputRefs = () => inputRefs;
-        const registerInputHandler = ref => {
-          if (ref !== null) {
-            inputRefs.push(ref);
-          }
-        };
-        viewOptions = { ...viewOptions, registerInputHandler };
-      }
+    // Fix scrollbar rendering issue in iOS 13+
+    // https://github.com/facebook/react-native/issues/26610#issuecomment-539843444
+    const scrollIndicatorInsets =
+      Platform.OS === 'ios' && parseInt(Platform.Version, 10) >= 13
+        ? { right: 1 }
+        : undefined;
 
-      props.showsHorizontalScrollIndicator = horizontal && showScrollIndicator;
-      props.showsVerticalScrollIndicator = !horizontal && showScrollIndicator;
-
-      const contentContainerStyleAttr = 'content-container-style';
-      if (this.props.element.getAttribute('scroll-orientation')) {
-        props.contentContainerStyle = createStyleProp(
-          this.props.element,
-          this.props.stylesheets,
-          {
-            ...viewOptions,
-            styleAttr: contentContainerStyleAttr,
-          },
-        );
-      }
+    // add sticky indicies
+    const stickyHeaderIndices = children.reduce(
+      (acc, element, index) =>
+        typeof element !== 'string' &&
+        element.props?.element?.getAttribute('sticky') === 'true'
+          ? [...acc, index]
+          : acc,
+      [],
+    );
 
-      // Fix scrollbar rendering issue in iOS 13+
-      // https://github.com/facebook/react-native/issues/26610#issuecomment-539843444
-      if (Platform.OS === 'ios' && parseInt(Platform.Version, 10) >= 13) {
-        props.scrollIndicatorInsets = { right: 1 };
-      }
+    return {
+      contentContainerStyle,
+      horizontal,
+      scrollIndicatorInsets,
+      showsHorizontalScrollIndicator: horizontal && showScrollIndicator,
+      showsVerticalScrollIndicator: !horizontal && showScrollIndicator,
+      stickyHeaderIndices,
+    };
+  };
 
-      if (horizontal) {
-        props.horizontal = true;
-      }
+  getScrollToInputAdditionalOffsetProp = (): number => {
+    const defaultOffset = 120;
+    if (this.attributes[ATTRIBUTES.SCROLL_TO_INPUT_OFFSET]) {
+      const offset = parseInt(
+        this.attributes[ATTRIBUTES.SCROLL_TO_INPUT_OFFSET],
+        10,
+      );
+      return Number.isNaN(offset) ? 0 : defaultOffset;
```

Comment:
do we not care about the actual value of offset?

```suggestion
      );
      return Number.isNaN(offset) ? defaultOffset : offset;

Reply:
hmm, you're right, this code seems odd. I rewrote it without changing the actual logic (see [here](https://github.com/Instawork/hyperview/pull/475/files#diff-d20a18a04fc2288d54bd648885b73443cbd01e36ba101bdc8331e4bdf9283437L93-L96)), but this might have been an existing bug.